Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -181,7 +182,7 @@ class EdgeHitProcessor: HitProcessing {
         }
     }
 
-    /// Extracts all the Edge configuration keys from the Configuration shared state
+    /// Extracts all the Edge configuration keys needed for this request. Configuration keys are extracted from the Configuration shared state and the given `event` instance.
```

Comment:
I recommend not mixing config keys with event keys in the same payload here to keep this function code specialized on a single item at a time. We could instead use a new helper `getRequestProperties(event)` that extract the request path overwrite and maybe additional request settings in the future.

Reply:
Additionally getEdgeConfig is used for all the endpoints we integrate with (currently interact and consent), we should allow the path overwrite only in the edge requestContent (interact) requests, and not for consent. Let's also include a test that overwrites are ignored for consent

Reply:
Should we revert this doc change?

Reply:
Don't forget to revert this doc change.